guides/source/api_app.md

```diff
@@ -374,7 +374,7 @@ controller modules by default:
 - `ActionController::Renderers::All`: Support for `render :json` and friends.
 - `ActionController::ConditionalGet`: Support for `stale?`.
 - `ActionController::BasicImplicitRender`: Makes sure to return an empty response, if there isn't an explicit one.
-- `ActionController::StrongParameters`: Support for parameters white-listing in combination with Active Model mass assignment.
+- `ActionController::StrongParameters`: Support for parameters permissions in combination with Active Model mass assignment.
```

Comment:
Can you change this to "Support for _parameter permissions_ in combination with Active Model mass assignment."?

Reply:
I think "filtering" might better convey what it actually does

Reply:
Ah, I agree. Let's go with "Support for _parameter filtering_ in combination with Active Model mass assignment." instead.

Reply:
Thanks, sounds much better!